pkg/model/stack.go

```diff
@@ -7,38 +7,32 @@ import (
 
 // swagger:model Stack
 type Stack struct {
-	CreatedAt          time.Time                `json:"createdAt"`
-	UpdatedAt          time.Time                `json:"updatedAt"`
-	Name               string                   `json:"name" gorm:"primaryKey"`
-	RequiredParameters []StackRequiredParameter `json:"requiredParameters" gorm:"foreignKey:StackName; references: Name; constraint:OnUpdate:CASCADE,OnDelete:CASCADE"`
-	OptionalParameters []StackOptionalParameter `json:"optionalParameters" gorm:"foreignKey:StackName; references: Name; constraint:OnUpdate:CASCADE,OnDelete:CASCADE"`
-	Instances          []Instance               `json:"instances" gorm:"constraint:OnUpdate:CASCADE,OnDelete:CASCADE;"`
-	HostnamePattern    string                   `json:"hostnamePattern"`
-	HostnameVariable   string                   `json:"hostnameVariable"`
+	Name             string           `json:"name" gorm:"primaryKey"`
+	Parameters       []StackParameter `json:"parameters" gorm:"foreignKey:StackName; references: Name; constraint:OnUpdate:CASCADE,OnDelete:CASCADE"`
+	Instances        []Instance       `json:"instances" gorm:"constraint:OnUpdate:CASCADE,OnDelete:CASCADE;"`
+	HostnamePattern  string           `json:"hostnamePattern"`
+	HostnameVariable string           `json:"hostnameVariable"`
+
+	CreatedAt time.Time `json:"createdAt"`
+	UpdatedAt time.Time `json:"updatedAt"`
```

Comment:
Moving these Gorm fields to the bottom kind of breaks the "convention" of having them at the top like we do for the other models. I'm okay either way, but just have it consistent.